### PR TITLE
RPC Web Socket - Example JS/HTML Page

### PR DESCRIPTION
**Pull request by miles200**
-Add an example JS/HTML page for utilising WebSocket with AdminRPC 

Requires the latest code from SEGS/develop and AuthServer running

### CODE REVIEW

File: Examples/RPCWebSocketTest.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  width: 150px;
+  height: 40px;
+}
+
+</style>
+
+<body>
```
Comment (dracc) [inline Examples/RPCWebSocketTest.html:35]: Pretty unconventional to put `<body>` inside `<head>`, normally this section would be below `</head>`, hence the names. :)

File: Examples/RPCWebSocketTest.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<HTML>
+<head>
+  <meta charset="utf-8" />
+  <title>SEGS RPC / WebSocket Test</title>
+
+<style>
```
Comment (dracc) [inline Examples/RPCWebSocketTest.html]: What happened to the indentation?

**Review by dracc — CHANGES_REQUESTED** [review]
Please fix the indentation and move `<body>`, apart from that it looks mighty fine. Great work @miles200!

**Comment by miles200** [comment]
@dracc Hey, I've made these changes would you mind checking? Thanks!

**Comment by dracc** [comment]
Nitpicking: `<script>` tag outside of `<head>`. Doesn't really matter as this will mostly be instructional for how to implement RPC towards the server.

Looks to me like this is ready to be merged. Great job @miles200 !

**Pull request merged**